code/datums/components/crafting/recipes.dm

```diff
@@ -305,6 +305,13 @@
 	category = CAT_STRUCTURES
 	time = 15 SECONDS
 
+/datum/crafting_recipe/iv_drip
+	name = "IV drip"
+	reqs = 	list(/obj/item/stack/rods = 4, /obj/item/stack/sheet/metal = 1)
+	result = /obj/machinery/iv_drip
+	category = CAT_STRUCTURES
+	time = 5 SECONDS
```

Comment:
should probably require a welding tool or screwdriver, and maybe some plastic

Reply:
screwdriver is alright